content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -109,15 +109,42 @@
                 that._elements["button"] = Array.isArray(that._elements["button"]) ? that._elements["button"] : [that._elements["button"]];
                 that._elements["panel"] = Array.isArray(that._elements["panel"]) ? that._elements["panel"] : [that._elements["panel"]];
 
-                if (that._properties.singleExpansion) {
-                    var expandedItems = getExpandedItems();
-                    // no expanded item annotated, force the first item to display.
-                    if (expandedItems.length === 0) {
-                        toggle(0);
+                // Expand the item based on deep-link-id if it matches with any existing accordion item id
+                var deepLinkIdIndex = -1;
+                if (config.deepLinkId) {
```

Comment:
```suggestion
                if (config.deepLinkId && config.deepLinkId.substring("-panel-") !== -1) {
```

Reply:
@bpauli : Do we need to make this change or else you are taking care of it?